.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,9 @@
+## What is the issue?
+
+## How do we reproduce this issue?
+
+## About your environment
+ - [ ] Self-hosted
+ - [ ] Python 3.5+
```

Comment:
Leaving `3.5+` in the template isn't helpful; it implies that `3.5+` tells us enough about the version to be able to debug the issue. (If they're using 3.6, we want to know about it.) `3.5` would be much better.